packages/synapse-react-client/src/components/SynapseTable/RowSelectionControls.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { CustomControl } from './TopLevelControls'
+import { QueryResultBundle, Row } from '@sage-bionetworks/synapse-types'
+import { Badge, Box, Button, Paper, Typography } from '@mui/material'
+
+export type RowSelectionControlsProps = {
+  customControls: CustomControl[]
+  data?: QueryResultBundle
+  selectedRows: Row[]
+  setSelectedRows: (newState: Row[]) => void
+  refresh: () => void
+}
+
+export const RowSelectionControls: React.FunctionComponent<
```

Comment:
New UI that is shown when there is non-empty a row selection (and custom commands)